resources/views/layouts/dashboard.blade.php

```diff
@@ -101,21 +97,25 @@
         </div>
     </nav>
 
-    <!-- success -->
+    <!-- success/failure -->
     @if ($message = Session::get('success'))
     <div class="alert alert-success">
         <p>{{ $message }}</p>
     </div>
     @endif
+    @elseif ($message = Session::get('failure'))
+    <div class="alert alert-danger">
+        <p>{{ $message }}</p>
+    </div>
```

Comment:
The if statement here is invalid and should be:
```php
<!-- success/failure -->
@if ($message = Session::get('success'))
    <div class="alert alert-success">
        <p>{{ $message }}</p>
    </div>
@elseif ($message = Session::get('failure'))
    <div class="alert alert-danger">
        <p>{{ $message }}</p>
    </div>
@endif
````

Reply:
fixed in 68b5a01